client/src/pages/SearchResultPage.vue

```diff
@@ -110,7 +114,12 @@ export default {
 
 				// Format results into object keyed by record_type
 				const resultFormatted = res.data.data.reduce((acc, cur) => {
-					return { ...acc, [cur.record_type]: cur };
+					return {
+						...acc,
+						[cur.record_type]: Array.isArray(acc[cur.record_type])
+							? [...acc[cur.record_type], cur]
+							: [cur],
+					};
```

Comment:
@maxachis see this part of the code wrt bug fix for #217. The previous logic was keying an object by `record_type` but forgot to take into account that there might be multiple records of the same time. To fix this bug, we're instead assigning an array to each of those keys, to accommodate multiple values of each key. The updates to the `template` above are in service of this change (as well as some general cleanup, as we're moving away from the `GridContainer` and `GridItem` components).

Reply:
Ah, that would make sense. So regardless of whether there's one record or multiple, it is formatted into an array, simplifying the logic on the backend.

Reply:
> Ah, that would make sense. So regardless of whether there's one record or multiple, it is formatted into an array, simplifying the logic on the backend.

Yeah @maxachis I think @mbodeantor and I are aligned on the idea that the API should be as simple as possible, and any complex manipulation required by UX or design requirements should be handled by the client.